src/components/Login/Login.js

```diff
@@ -0,0 +1,83 @@
+import React, { Component } from 'react';
+import './Login.css';
+
+class Login extends Component {
+  constructor() {
+    super();
+    this.state = {
+      name: '',
+      email: '',
+      purpose: '',
+      error: ''
+    }
+  }
+
+  updateNameState = (event) => {
+    this.setState({name: event.target.value})
+  }
+
+  updateEmailState = (event) => {
+    this.setState({email: event.target.value})
+  }
+
+  updatePurposeState = (event) => {
+    this.setState({email: event.target.value})
+  }
+
+  onLogin = (event) => {
+    event.preventDefault();
+    if (!this.state.name || !this.state.email || !this.state.purpose) {
+      this.setState({error: '*All input fields required'})
+    }
+  }
+
+  render() {
+    return (
+      <form>
+        <label>Name</label>
+        <input 
+          type='text'
+          placeholder=''
+          value={this.state.name}
+          onChange={this.updateNameState}>
+        </input>
+        <label>Email</label>
+        <input
+          type='text'
+          placeholder=''
+          value={this.state.email}
+          onChange={this.updateEmailState}>  
+        </input>
+
+        <input 
+          type="radio" 
+          name="purpose" 
+          value={this.state.purpose}
```

Comment:
the value for this one should be "business", and the other two should be "vacation" and "other", I would also probably change the name properties to match as well